packages/web-extension/src/messaging/BackgroundMessenger.ts

```diff
@@ -22,7 +22,7 @@ export const createBackgroundMessenger = ({ logger, runtime }: MessengerDependen
   const getChannel = (channelName: ChannelName) => {
     let channel = channels.get(channelName);
     if (!channel) {
-      channels.set(channelName, (channel = { message$: new Subject(), ports$: new BehaviorSubject(new Set()) }));
+      channels.set(channelName, (channel = { message$: new ReplaySubject(1), ports$: new BehaviorSubject(new Set()) }));
```

Comment:
Given this is a fix we're not completely satisfied with, please add a comment inline to ensure there's some context to an otherwise curious need to use this special `Subject`